tensorflow/compiler/tf2tensorrt/utils/trt_shape_optimization_profiles_test.cc

```diff
@@ -125,9 +154,19 @@ class TrtShapeOptimizationProfileTest : public ::testing::Test {
       1U << static_cast<int>(
           nvinfer1::NetworkDefinitionCreationFlag::kEXPLICIT_BATCH);
 #endif
+  ProfileStrategy strategy_;
 };
 
-TEST_F(TrtShapeOptimizationProfileTest, Static) {
+INSTANTIATE_TEST_CASE_P(
+    OptProfilesTestInstantiation, TrtShapeOptimizationProfileTest,
+    ::testing::Values(ProfileStrategy::kRange, ProfileStrategy::kOptimal,
+                      ProfileStrategy::kRangeOptimal,
+                      ProfileStrategy::kImplicitBatchModeCompatible));
+
+TEST_P(TrtShapeOptimizationProfileTest, Static) {
+  // Static mode does not depend on strategies, we only once.
```

Comment:
Sorry that I don't understand this comment. What is "we only once"?

Reply:
Hmm, the verb got deleted from the sentence. Fixed it.